person.rb

```diff
@@ -26,4 +30,15 @@ def is_of_age?
   def can_use_services?
     is_of_age? || @parent_permission
   end
+
+  def correct_name
+    @name
+  end
+
+  def rent_bool(date, book)
```

Comment:
- [ ] [OPTIONAL] Please consider renaming your method as I feel there is a typo there. You may have intended to name it `rent_book` instead of `rent_bool` 🤔